<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ricerca_file_window</class>
 <widget class="QMainWindow" name="Ricerca_file_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search file in system</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/MGrep.ico</normaloff>:/icons/icons/MGrep.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="11">
     <widget class="QPushButton" name="b_pathname">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/folder.gif</normaloff>:/icons/icons/folder.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="4">
     <widget class="QLabel" name="l_excludepath">
      <property name="text">
       <string>Exclude directories</string>
      </property>
      <property name="buddy">
       <cstring>e_excludepath</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="4">
     <widget class="QLabel" name="l_pathname_2">
      <property name="text">
       <string>Folder name</string>
      </property>
      <property name="buddy">
       <cstring>e_pathname</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="7">
     <widget class="QCheckBox" name="c_cache_file_system">
      <property name="text">
       <string>Clear cache file system</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="8" colspan="3">
     <widget class="QLineEdit" name="e_excludepath"/>
    </item>
    <item row="0" column="8" colspan="3">
     <widget class="QLineEdit" name="e_pathname"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="e_filesearch"/>
    </item>
    <item row="3" column="11">
     <widget class="QPushButton" name="b_excludepath">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/folder.gif</normaloff>:/icons/icons/folder.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="l_filesearch">
      <property name="text">
       <string>File search</string>
      </property>
      <property name="buddy">
       <cstring>e_filesearch</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="l_filter">
      <property name="text">
       <string>File filter</string>
      </property>
      <property name="buddy">
       <cstring>e_filter</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="5" colspan="3">
     <widget class="QPushButton" name="b_add_line">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add the selected line to 'My favorites files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/add.gif</normaloff>:/icons/icons/add.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="b_save_pref">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save the search data in order to be retrieved at the next start of SmiGrep&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/disk.gif</normaloff>:/icons/icons/disk.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="8" rowspan="2" colspan="3">
     <widget class="QLineEdit" name="e_filter"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="l_risultati">
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="14">
     <widget class="QListView" name="o_lst1">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="7" column="8" colspan="4">
     <widget class="QPushButton" name="b_search">
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Start search</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/go.gif</normaloff>:/icons/icons/go.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>e_filesearch</tabstop>
  <tabstop>e_pathname</tabstop>
  <tabstop>e_filter</tabstop>
  <tabstop>e_excludepath</tabstop>
  <tabstop>b_search</tabstop>
  <tabstop>o_lst1</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>o_lst1</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Ricerca_file_window</receiver>
   <slot>o_lst1_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>773</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_save_pref</sender>
   <signal>clicked()</signal>
   <receiver>Ricerca_file_window</receiver>
   <slot>b_save_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_add_line</sender>
   <signal>clicked()</signal>
   <receiver>Ricerca_file_window</receiver>
   <slot>b_add_line_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_pathname</sender>
   <signal>clicked()</signal>
   <receiver>Ricerca_file_window</receiver>
   <slot>b_pathname_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_excludepath</sender>
   <signal>clicked()</signal>
   <receiver>Ricerca_file_window</receiver>
   <slot>b_excludepath_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_search</sender>
   <signal>clicked()</signal>
   <receiver>Ricerca_file_window</receiver>
   <slot>b_search_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>957</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>b_pathname_slot()</slot>
  <slot>b_save_slot()</slot>
  <slot>b_excludepath_slot()</slot>
  <slot>b_search_slot()</slot>
  <slot>b_add_line_slot()</slot>
  <slot>o_lst1_slot()</slot>
 </slots>
</ui>
